internal/i18n/en_translations.go

```diff
@@ -18,205 +18,234 @@ package i18n
+	MsgContractWrongInputType       = ffm("FF10305", "Input '%v' is of type '%v' not expected type of '%v'", 400)
+	MsgContractMissingInputField    = ffm("FF10306", "Expected object of type '%v' to contain field named '%v' but it was missing", 400)
+	MsgContractMapInputType         = ffm("FF10307", "Unable to map input type '%v' to known FireFly type - was expecting '%v'", 400)
+	MsgContractByteDecode           = ffm("FF10308", "Unable to decode field '%v' as bytes", 400)
+	MsgContractInternalType         = ffm("FF10309", "Input '%v' of type '%v' is not compatible blockchain internalType of '%v'", 400)
+	MsgContractLocationInvalid      = ffm("FF10310", "Failed to validate contract location: %v", 400)
+	MsgContractParamInvalid         = ffm("FF10311", "Failed to validate contract param: %v", 400)
+	MsgContractSubscriptionExists   = ffm("FF10312", "A contract subscription already exists in the namespace: '%s' with name: '%s'", 409)
+	MsgContractMethodNotSet         = ffm("FF10313", "Method not specified on invoke contract request", 400)
+	MsgContractNoMethodSignature    = ffm("FF10314", "Method signature is required if interfaceID is absent", 400)
+	MsgContractMethodResolveError   = ffm("FF10315", "Unable to resolve contract method: %s", 400)
+	MsgContractLocationExists       = ffm("FF10316", "The contract location cannot be changed after it is created", 400)
+	MsgSubscriptionNoEvent          = ffm("FF10317", "An eventId or in-line event definition must be supplied when subscribing", 400)
+	MsgSubscriptionEventNotFound    = ffm("FF10318", "No event was found in namespace '%s' with id '%s'", 400)
+	MsgEventNameMustBeSet           = ffm("FF10319", "Event name must be set", 400)
+	MsgMethodNameMustBeSet          = ffm("FF10320", "Method name must be set", 400)
+	MsgContractEventResolveError    = ffm("FF10321", "Unable to resolve contract event", 400)
+	MsgQueryOpUnsupportedMod        = ffm("FF10322", "Operation '%s' on '%s' does not support modifiers", 400)
+	MsgDXBadSize                    = ffm("FF10323", "Unexpected size returned from data exchange upload. Size=%d Expected=%d")
+	MsgBlobMismatchSealingData      = ffm("FF10324", "Blob mismatch when sealing data")
+	MsgFieldTypeNoStringMatching    = ffm("FF10325", "Field '%s' of type '%s' does not support partial or case-insensitive string matching", 400)
+	MsgFieldMatchNoNull             = ffm("FF10326", "Comparison operator for field '%s' cannot accept a null value", 400)
+	MsgTooLargeBroadcast            = ffm("FF10327", "Message size %.2fkb is too large for the max broadcast batch size of %.2fkb", 400)
+	MsgTooLargePrivate              = ffm("FF10328", "Message size %.2fkb is too large for the max private message size of %.2fkb", 400)
+	MsgManifestMismatch             = ffm("FF10329", "Manifest mismatch overriding '%s' status as failure: '%s'", 400)
+	MsgWSHeartbeatTimeout           = ffm("FF10330", "Websocket heartbeat timed out after %.2fms", 500)
```

Comment:
😓

Reply:
I think some additional strings were pulled in beyond what's actually needed for this change... is that intentional?